src/components/HighlightContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+import { colors, dropShadow } from "src/common/constants";
+
+interface IHighlightContainer {
+  flexDirection?: string,
+  justifyContent?: string,
+  alignItems?: string
+}
+
+const HighlightContainer = styled.div<IHighlightContainer>`
```

Comment:
I don't think that this would be re-used. Let's move it to the file where it's used.

Reply:
HighlightContainer would be reused multiple times. 
It's the ~blue box. Will be re-used.